nservicebus/messaging/transactions.md

```diff
@@ -118,12 +106,10 @@ NOTE: This requires the selected storage to support enlisting in transaction sco
 
 WARNING: This might escalate to a distributed transaction if data in different databases are updated.
 
-
 ## Controlling transaction scope options
 
 The following options for transaction scopes used during message processing can be configured.
 
```

Comment:
Fixed

On Mon, Dec 21, 2015 at 11:58 PM, Simon Cropp <notifications@github.com>
wrote:

> In nservicebus/messaging/transactions.md
> <https://github.com/Particular/docs.particular.net/pull/988#discussion_r48203046>
> :
>
> > @@ -118,12 +106,10 @@ NOTE: This requires the selected storage to support enlisting in transaction sco
> >
> >  WARNING: This might escalate to a distributed transaction if data in different databases are updated.
> >
> > -
> >  ## Controlling transaction scope options
> >
> >  The following options for transaction scopes used during message processing can be configured.
> >
>
> Why the whitespace removal. Some spacing between headers makes the raw md
> much more readable
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Particular/docs.particular.net/pull/988/files#r48203046>
> .
>